style.css

```diff
@@ -74,6 +74,200 @@ nav a:hover {
   color: #505f79;
 }
 
+.icons li a {
+  color: #505f79;
+}
+
 .headline li a:hover {
   cursor: pointer;
 }
+
+.portfolio {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  margin: 70px 3.2%;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 4.5%;
+  background-color: #fff;
+  border-radius: 4.5%;
+  gap: 12px;
+  margin: 44px 0;
+}
+
+.card h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.card p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.xp {
+  display: flex;
+  align-items: center;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 8px;
+}
+
+.xp p {
+  color: #344563;
+  font-weight: 600;
+}
+
+.xp ul {
+  display: flex;
+}
+
+.xp li {
+  color: #7a869a;
+  margin-left: 16px;
+}
+
+.xp li::before {
+  content: "";
+  display: inline-block;
+  margin-bottom: 1px;
+  width: 8px;
+  height: 8px;
+  margin-right: 16px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.skills {
+  display: flex;
+  margin-bottom: 16px;
+}
+
+.sbox {
+  background: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin-right: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.btn {
+  background-color: #fff;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  width: 126px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  text-align: center;
+  cursor: pointer;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.btn:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-top-right-radius: 60px;
+  padding: 114px 6.5%;
+  position: relative;
+  z-index: 1;
+}
```

Comment:
- [ ] The background color of `about` section in the template is has a rounded-right corner and according to the requirement your design should be as close to the template as possible.  Kindly make right corner of about  section `rounded. 

| Figma design | Your design |
|--------------|--------------|
|<img width="301" alt="Screenshot 2023-04-29 at 1 55 35 PM" src="https://user-images.githubusercontent.com/44798044/235294952-489301a3-0bc3-4055-9b93-99b6454c7828.png">|<img width="366" alt="Screenshot 2023-04-29 at 1 58 55 PM" src="https://user-images.githubusercontent.com/44798044/235294957-c6e9ee04-c36c-48ba-bf0b-68704fa7a0af.png">|

Reply:
Also added rounded shape as required by reviewer.